docker/Dockerfile

```diff
@@ -36,13 +36,12 @@ ENV LANG en_US.utf8
 
 # Install foundry from specific commit
 SHELL ["/bin/bash", "-c"]
-RUN curl -L https://raw.githubusercontent.com/foundry-rs/foundry/037b3bc9cebd545fe3a4f05a32bf7efb82afdbd8/foundryup/install > /tmp/foundry-install-script && \
-    sed -i 's/\/foundry-rs\/foundry\/master\/foundryup/\/foundry-rs\/foundry\/037b3bc9cebd545fe3a4f05a32bf7efb82afdbd8\/foundryup/g' /tmp/foundry-install-script && \
+RUN curl -L https://raw.githubusercontent.com/ava-labs/foundry/master/foundryup/install > /tmp/foundry-install-script && \
```

Comment:
Still not in love with the fact that changes to the `foundryup` script in the primary branch will get pulled in without an explicit update. Could we use the `v0.1.0` tag for that too?

Reply:
Yup, good call

Reply:
To the same end, we should probably keep the `sed` statement to use the specific commit/tag here: https://github.com/ava-labs/foundry/blob/v0.1.0/foundryup/install#L11

Reply:
Or if you don't think that's necessary, would recommend piping the CURL output directly into `bash` rather than writing to a temp file.